### PR TITLE
chsh: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/chsh.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # chsh
 
-> Change the user's login shell.
+> Change a user's login shell.
 > More information: <https://manned.org/chsh>.
 
-- Change the current user's login shell interactively:
+- Set a specific login shell for a current user interactively:
```
Comment (mfrw) [inline pages/common/chsh.md:6]: maybe `the` sounds better .. but agian .. I am not a grammar expert 
```suggestion
- Set a specific login shell for the current user interactively:
```

File: pages/common/chsh.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 `chsh`
 
-- Change the login shell of the current user:
+- Set a specific login shell for a current user:
```
Comment (mfrw) [inline pages/common/chsh.md:10]:
```suggestion
- Set a specific login shell for the current user:
```

File: pages/linux/chsh.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
 # chsh
 
-> Change the user's login shell.
+> Change a user's login shell.
 > More information: <https://manned.org/chsh>.
 
-- Change the current user's login shell interactively:
+- Set a specific login shell for a current user interactively:
```
Comment (mfrw) [inline pages/linux/chsh.md:6]:
```suggestion
- Set a specific login shell for the current user interactively:
```

File: pages/linux/chsh.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 
 `chsh`
 
-- Change the current user's login shell:
+- Set a specific login shell for a current user:
```
Comment (mfrw) [inline pages/linux/chsh.md:10]:
```suggestion
- Set a specific login shell for the current user:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
`--list-shells` isn't portable.

**Review by kbdharun — APPROVED** [review]

File: pages/common/chsh.md [hunk 5]
```diff
@@ -1,20 +1,20 @@
 # chsh
 
-> Change the user's login shell.
+> Change a user's login shell.
```
Comment (mfrw) [inline pages/common/chsh.md:3]: How about we do not put an article here:
`Change user's login shell` ?

Comment (mfrw) [inline pages/common/chsh.md:3]: How about we drop the article: ~a/the~

Just like we have for `[l]ist available shells` :)
```suggestion
> Change user's login shell.
```

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @EmilySeville7cfg

**Pull request merged**